packages/actor-bindings-aggregator-factory-average/test/AverageAggregator-test.ts

```diff
@@ -17,8 +16,9 @@ import {
   getMockEEFactory,
   int,
   makeAggregate,
-} from '@comunica/utils-jest';
+} from '@comunica/utils-expression-evaluator/test/util/helpers';
```

Comment:
I'll stop here with checking the imports, as many of them seem incorrect 😅

Reply:
this happens in a test file, so its ok